### PR TITLE
Third pass on coding style

### PR DESCRIPTION
**Pull request by mosteo**
I've enabled -gnatDIO, only O needed fixes.

@Fabien-Chouteau, now that the style is fixed I'd prefer that the checks be enabled by default; your thoughts?

### CODE REVIEW

**Comment by pmderodat** [comment]
Sorry if I forgot guys, but what do you think about using GNAT’s own style checks here? (`-gnatyg`)

Also I’m all for enabling the checks once the style is fixed :) Thank you both for your work on this!

**Comment by mosteo** [comment]
I've tried -gnatyg and it only complains about multiple blank lines. I guess these can be improved by adding a comment about what that extra separation intended to draw attention to, if anything.

So, if no objection from @Fabien-Chouteau, I'm going to enable and fix it in this PR.

**Comment by mosteo** [comment]
I finally didn't use -gnatyD, since it doesn't allow acronyms.

File: src/alire/alire-actions.adb [hunk 1]
```diff
@@ -17,7 +17,7 @@ package body Alire.Actions is
    -- To_TOML --
    -------------
 
-   function To_TOML (This : Run) return TOML.TOML_Value is
+   overriding function To_TOML (This : Run) return TOML.TOML_Value is
```
Comment (Fabien-Chouteau) [inline src/alire/alire-actions.adb]: I always put `overriding` on the line above `function`.

Comment (mosteo) [reply src/alire/alire-actions.adb]: OK, will do in the future, thanks.

**Pull request merged**